modules/manual-configuration-image-registry-qcr.adoc

```diff
@@ -5,21 +5,35 @@
 [id="manual-configuration-image-registry-qcr_{context}"]
 = Manually configuring Quay Container Registry
 
-You can integrate {product-title} with Quay Container Registry.
+You can integrate {product-title} ({product-title-short}) with Quay Container Registry. You can integrate with Quay using the following methods:
+
+- Integrating with the Quay public repository (registry): This method does not require authentication.
+- Integrating with a Quay private registry using a robot account: This method requires that you create a robot account to use with Quay (recommended). See the link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/use_red_hat_quay/use-quay-manage-repo#allow-robot-access-user-repo[Quay documentation] for more information.
+- Integrating with Quay to use the Quay scanner rather than the {product-title-short} scanner:  This method uses the API and requires an OAuth token for authentication.
 
 .Prerequisites
-* You must have an OAuth token for authentication with the Quay Container Registry.
+* For authentication with a Quay private registry, you need the credentials associated with a robot account or an OAuth token (deprecated).
+* To configure an integration to use the Quay scanner instead of the {product-title-short} scanner, you need an OAuth token.
 
 .Procedure
-. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. On the {product-title-short} portal, navigate to *Platform Configuration* -> *Integrations*.
 . Under the *Image Integrations* section, select *Red Hat Quay.io*.
 . Click *New integration*.
-. Enter the details for the following fields:
-.. *Integration name*: The name of the integration.
-.. *Type*: Select *Registry*.
-.. *Endpoint*: The address of the registry.
-.. *OAuth Token*
-. If you are not using a TLS certificate when connecting to the registry, select *Disable TLS certificate validation (insecure)*.
-. Select *Create integration without testing* to create the integration without testing the connection to the registry.
-. Select *Test* to test that the integration with the selected registry is working.
+. Enter the *Integration name.*
+. Enter the *Endpoint*, or the address of the registry.
+.. If you are integrating with the Quay public repository, under *Type*, select *Registry*, then go to Step 6. 
+.. If you are integrating with a Quay private registry, under *Type*, select *Registry* and enter information in the following fields:
+** *Robot username*: If you are using a Quay robot account to access the registry, enter the user name in the format `_<namespace><accountname>_`.
```

Comment:
Can you double-check if a plus separated namespace from accountname in the user name?

https://docs.quay.io/glossary/robot-accounts.html

Reply:
Yea it's plus separated. I'm not sure if it's worth calling out separately since Quay enforces that all created robot accounts have it anyway.